src/core/shape_inference/include/glu_with_clamp_shape_inference.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "intel_gpu/op/swiglu_with_clamp.hpp"
+#include "utils.hpp"
+#include "variadic_split_shape_inference.hpp"
+
+namespace ov {
+namespace op {
+namespace internal {
```

Comment:
Please move this to gpu op, because currently only gpu internal op SwiGluWithClamp has this feature